libsolidity/analysis/StaticAnalyzer.cpp

```diff
@@ -21,14 +21,16 @@
  */
 
 #include <libsolidity/analysis/StaticAnalyzer.h>
-#include <memory>
 #include <libsolidity/ast/AST.h>
+#include <libsolidity/inlineasm/AsmAnalysis.h>
+#include <libsolidity/inlineasm/AsmAnalysisInfo.h>
+#include <libsolidity/inlineasm/AsmData.h>
```

Comment:
Realised that none of these includes are needed now I think?

@roadriverrail could you remove in a subsequent PR?